scripts/table_diff/changes/changes.go

```diff
@@ -105,19 +126,33 @@ func getColumnChanges(file *gitdiff.File, table string) (changes []change) {
 				})
 			}
 
-			if addedColumn.pk && !deletedColumn.pk {
+			if addedColumn.pk() && !deletedColumn.pk() {
 				changes = append(changes, change{
 					Text:     fmt.Sprintf("Table %s: primary key constraint added to column %s", backtickStrings(table, deletedName)...),
 					Breaking: false,
 				})
 			}
 
-			if !addedColumn.pk && deletedColumn.pk {
+			if !addedColumn.pk() && deletedColumn.pk() {
 				changes = append(changes, change{
 					Text:     fmt.Sprintf("Table %s: primary key constraint removed from column %s", backtickStrings(table, deletedName)...),
 					Breaking: false,
 				})
 			}
+
+			if addedColumn.incremental() && !deletedColumn.incremental() {
+				changes = append(changes, change{
+					Text:     fmt.Sprintf("Table %s: incremental sync support added to column %s", backtickStrings(table, deletedName)...),
+					Breaking: false,
```

Comment:
We may want to make this breaking, as there's a chance an update to the cursor or a full sync will be necessary on upgrade. Debatable, because it could simply be for documentation purposes or backwards-compatibility could be handled in the code. But maybe this comment can call it to attention and then we can decide on a case-by-case basis?

Reply:
Sure, let's make it breaking and we can change it it doesn't make sense